### PR TITLE
style: top part be consistent in mobile

### LINKED ISSUE #6986: Consistency across mobile burger items and sidebars
image
image

- I would like to see the top part being consistent across all the items in the burgermenu
- currently, its not like this for cart, profile, notification 
image

- changing the top part on devices with collapsed navbar

### PR DESCRIPTION
**Pull request by xiiiAtCn**
## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Closes #6986 
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [ ] My DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=12jXHf7yYF4wTRjDPbwFB24V1nVRPRkuU2S8Ke5ZWTfvyBGW&usdamount=8.82&donation=true)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at 397ed34</samp>

This pull request refactors several modal components to use a common `NeoModalHead` component from the ui library, which adapts to different devices and improves code reusability and consistency. The `NeoModalHead` component is defined in `NeoModalHead.vue` and `NeoModalHead.scss` files, and exported from the `index.ts` file. The custom styles for the modal headers are removed from the component-specific style files.

### <samp>🤖 Generated by Copilot at 397ed34</samp>

> _Sing, O Muse, of the skillful coder who refashioned_
> _The modal heads of many components, using `NeoModalHead`,_
> _A wondrous work of art, responsive and adaptable,_
> _Like the shape-shifting Proteus, who eludes all capture._

### CODE REVIEW

File: libs/ui/src/components/NeoModalHead/NeoModalHead.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+defineProps<{
+  title: string
+}>()
+
+const isMobile = ref(window.innerWidth < 1024)
+
+const emit = defineEmits(['close'])
+</script>
+
+<style lang="scss">
```
Comment (stephenjason89) [inline libs/ui/src/components/NeoModalHead/NeoModalHead.vue:33]: use style scoped

**Comment by stephenjason89** [comment]
code looks good to me

**Review by stephenjason89 — APPROVED** [review]

**Comment by exezbcz** [comment]
hello, nice job! 

just want to make sure you are not changing anything on the desktop sidebars 
deploy: 
image
canary: 
image


on mobile, there should be the right pointing chevron on cart, notification and profile. When you open this section, you can get to the burger menu list with the top part chevron. Right now the top part of the "desktop" sidebars closes the burger menu.

**Comment by prury** [comment]
across some of them: 
both dark & white mode: 

image

**Comment by xiiiAtCn** [comment]
> across some of them: both dark & white mode:
> 
> image 

@prury  I notice `Explore` and `Create`  panel has a high contrast title to its content. So I keep `notifications`,`cart` and `profile` consistent with them.  Shall we change this?

**Comment by exezbcz** [comment]
@prury no cross on mobile

@xiiiAtCn one thing left, there is this weird flash of the landing page when selecting items that are sidebar on the desktop

I have video but the preview is not working -_-

**Comment by xiiiAtCn** [comment]
> @xiiiAtCn one thing left, there is this weird flash of the landing page when selecting items that are sidebar on the desktop

when navigating to `notification`,  we closed the burger menu and then open a modal. so you saw the flash of the landing page. I can change this previous logic if needed.

**Comment by exezbcz** [comment]
@xiiiAtCn would be nice to fix that without affecting desktop sidebars and other functionalities

**Comment by xiiiAtCn** [comment]
> @xiiiAtCn would be nice to fix that without affecting desktop sidebars and other functionalities 

I have fixed this. Have a try

**Comment by exezbcz** [comment]
Thanks for fixing that. One small thing: when I add to cart an item from the gallery item and then go to the cart, the top part is not the same as on the mobile 

- Otherwise it is correct that when I click the chevron, I go back - to the gallery item 
image
image

**Comment by xiiiAtCn** [comment]
> Thanks for fixing that. One small thing: when I add to cart an item from the gallery item and then go to the cart, the top part is not the same as on the mobile

have fixed. The reason is page scroll will trigger top bar sretched and open burger menu can reset it however open modal will not

**Comment by exezbcz** [comment]
@prury could you test please sir? 👀

**Comment by prury** [comment]
> @prury could you test please sir? 👀

did some back and forth with profile, cart, notifications, connect wallet and it seems ok ✔️ 
desktop version seems non affected aswell ✔️

**Comment by exezbcz** [comment]
> > @prury could you test please sir? 👀
> 
> did some back and forth with profile, cart, notifications, connect wallet and it seems ok ✔️ 
> desktop version seems non affected aswell ✔️ 
> 

Thanks!

**Comment by exezbcz** [comment]
Nice job, thank you

**Comment by yangwao** [comment]
pay 50 usd

If the mobile navbar would be Ferrari, it's most fast iterated component in KodaDot

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 4.46 USD/DOT ~ 11.211 $DOT
🧗 [12jXHf7yYF4wTRjDPbwFB24V1nVRPRkuU2S8Ke5ZWTfvyBGW](https://polkadot.subscan.io/account/12jXHf7yYF4wTRjDPbwFB24V1nVRPRkuU2S8Ke5ZWTfvyBGW)
🔗 [0x5a475174aa6f10eab78a27505a7bb4e3b7fe9ddbfeba7c943cfe59e6be94bc6c](https://polkadot.subscan.io/extrinsic/0x5a475174aa6f10eab78a27505a7bb4e3b7fe9ddbfeba7c943cfe59e6be94bc6c)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by exezbcz** [comment]
@yangwao what color is your bugatti

**Pull request merged**